modules/ROOT/pages/getting-started.adoc

```diff
@@ -0,0 +1,127 @@
+== Fedora CoreOS - Getting Started
+
+Fedora CoreOS has no install-time configuration. All Fedora CoreOS installs or cloud images start the same, then on first boot Ignition will read the supplied config and configure the system. Ignition configs are usually supplied via the cloud’s userdata mechanism, or, in the case of bare metal, injected at install time. This guide will show you how to launch Fedora CoreOS on AWS, QEMU, and bare metal as well as how to create Ignition configs.
+
+=== Launching Fedora CoreOS
+
+==== Booting on AWS:
+
+To boot on AWS, find the correct ami in the https://builds.coreos.fedoraproject.org/browser[release browser] and specify the Ignition config as the https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html#instancedata-add-user-data[user-data]. Note that while the AWS documentation mentions cloud-init and scripts, Fedora CoreOS only accepts Ignition configs; it does not support cloud-init or running scripts from userdata.
+
+. Example launching Fedora CoreOS on AWS
+[source, bash]
+aws ec2 run-instances <other options> --user-data file://config.ign 
+
+==== Booting with QEMU:
+
+Download the QEMU Fedora CoreOS image from the https://builds.coreos.fedoraproject.org/browser[release browser]
+
+Specify your Ignition config by using `-fw_cfg` to set the `opt/com.coreos/config` key in the QEMU firmware configuration device.
+
+. Example launching Fedora CoreOS with QEMU
+[source, bash]
+qemu-system-x86_64 -machine accel=kvm -m 2048 -cpu host -nographic \
+	-drive if=virtio,file=path/to/fedora-coreos-qemu.qcow2 \
+	-device virtio-rng-pci \
+	-fw_cfg name=opt/com.coreos/config,file=path/to/ignition-config.ign
+
+==== Installing on bare metal:
+
+Follow the https://github.com/coreos/coreos-installer/[coreos-installer instructions] to install Fedora CoreOS to disk. This will inject your specified Ignition config in the installed image. Note that Fedora CoreOS does not support running live PXE booted yet.
+
+=== Generating Ignition configs
+
+Ignition configs are not intended to be hand-written. Instead, use the https://github.com/coreos/fcct[Fedora CoreOS Config Transpiler (FCCT)] to translate Fedora CoreOS Configs (FCCs) to Ignition configs. In addition to be easier to write, FCCT will also validate your config and check for errors. Here’s an example FCC that adds an SSH key to the `core` user’s authorized keys:
```

Comment:
Let's not lead with that first sentence.  Maybe just tell people what they _should_ do, and add "[easier to write] than Ignition configs"?

Reply:
I'm not sure if I agree. We _don't_ want people writing Ignition configs and I'd prefer to be up front with that. If we introduce FCCT after the fact it both seems more like a choice and FCCTs reason for existence isn't as clear.

Reply:
Okay, I can live with that.